SampleApps/ReceiverDemoSwift/ReceiverDemoSwift-Bridging-Header.h

```diff
@@ -0,0 +1,5 @@
+#import <DeepLinkKit/DeepLinkKit.h>
```

Comment:
Did you have to make this explicitly? If so, should we include it in DLK for all to use?